common-core-open/src/main/java/edu/isi/nlp/files/KeyValueSources.java

```diff
@@ -92,4 +93,26 @@ private KeyValueSources() {
     }
     return new ZipKeyValueSource(zipFile, ret.build());
   }
+
+  @Nonnull
+  public static ImmutableKeyValueSource<Symbol, ByteSource> fromVistaUtilsZipKeyValueSource(
```

Comment:
Javadoc, please

Reply:
In the Javadoc, note this will only work if this originated from a character or byte sink from https://github.com/isi-vista/vistautils/blob/0b0be9d842fb4d547b3e020a6b5687cc6105ca72/vistautils/key_value.py#L95 with default settings.